napari/_qt/qt_event_loop.py

```diff
@@ -139,13 +138,10 @@ def get_app(
     else:
         # automatically determine monitor DPI.
         # Note: this MUST be set before the QApplication is instantiated
-        if PYQT5:
-            QApplication.setAttribute(
-                Qt.ApplicationAttribute.AA_EnableHighDpiScaling
-            )
-            QApplication.setAttribute(
-                Qt.ApplicationAttribute.AA_UseHighDpiPixmaps
-            )
+        QApplication.setAttribute(
+            Qt.ApplicationAttribute.AA_EnableHighDpiScaling
+        )
+        QApplication.setAttribute(Qt.ApplicationAttribute.AA_UseHighDpiPixmaps)
```

Comment:
Do you have any idea why there was a if PYQT5 and it's not necessary anymore ? Just to make sure.

Reply:
Yes. It was removed in Qt6. 
But I do not know/remeber why it was introduced.

Reply:
Qt::AA_EnableHighDpiScaling was added in Qt5 (5.6) and has been removed in Qt6.2 (https://doc.qt.io/qtforpython/porting_from2.html)
So I think this does need to be within an `If` so we can (experimentally) support Qt6.